src/style.css

```diff
@@ -2,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf) format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 900;
+  font-style: normal;
+}
+
+.search__form {
+  margin-top: 20px;
```

Comment:
This is a good example of using top margins consistently, as per the checklist. Remember to be consistent with your margins and padding throughout your CSS. This will help to avoid potential margin collapse and make your CSS easier to understand and maintain.